pyproject.toml

```diff
@@ -12,18 +12,22 @@ Scrapy = "^2.7.0"
 scrapy-user-agents = "^0.1.1"
 # работа с MongoDB
 pymongo = {version = "^4.3.2", extras = ["srv"]}
+# валидация данных
+pydantic = "^1.10.2"
 
-[tool.poetry.dev-dependencies]
 # документация
+[tool.poetry.group.documentation.dependencies]
```

Comment:
make documentation, tests and linters groups optional

Reply:
Done